src/d3a/d3a_core/util.py

```diff
@@ -532,3 +530,7 @@ def _dispatch_tick_frequency(self) -> int:
 
     def is_it_time_for_external_tick(self, current_tick_in_slot) -> bool:
         return current_tick_in_slot % self._dispatch_tick_frequency == 0
+
+
+def should_read_profile_from_db(profile_uuid):
```

Comment:
How about moving it to `src/d3a/d3a_core/user_profile_handler.py` module?

Reply:
It's really a matter of taste as it is used in a lot of places (mainly in the strategies).
So I would wait for another one voting for it.